mvp-spec/operator-api.md

```diff
@@ -136,66 +210,81 @@ For details on how to calculate or verify signatures, see [signatures.md](signat
 - if `paf_preferences` cookie exists, return its values
 - [_on REST version only_] attempt to create a temporary, short-life, "test 3PC" cookie
 
-#### REST read: `GET /v1/id-prefs`
+#### REST read: `GET /v1/ids-prefs`
 
 | Message  | Format                                                  |
 |----------|---------------------------------------------------------|
-| Request  | [get-id-prefs-request](model/get-id-prefs-request.md)   |
-| Response | [get-id-prefs-response](model/get-id-prefs-response.md) |
+| Request  | [get-ids-prefs-request](model/get-ids-prefs-request.md)   |
+| Response | [get-ids-prefs-response](model/get-ids-prefs-response.md) |
 
-##### Example
+<details>
+<summary>Full example</summary>
```

Comment:
In the REST variant, we're proposing a GET call with a body. 

This puts a dependency on any web caching system not to use only the URI as caching key. In a somewhat distant past this wasn't a safe assumption. I'm not sure how safe this assumption can be today.

The 2 safe options are: either use GET with query string parameters but no body, OR use POST with a body

see https://stackoverflow.com/questions/978061/http-get-with-request-body

Reply:
I must misunderstand your comment because the REST variant never uses body with GET.
Now only the POST sets a body (and no query string). In the past the POST was also setting some query string parameters, which I realized was not very convenient, so this PR moves every thing to the payload for the POST endpoint.
But for all other endpoints using GET, nothing is in the payload.
So I don't follow you.

Reply:
ok i misunderstood, no issue here